runtime/compiler/control/J9Options.cpp

```diff
@@ -772,6 +774,10 @@ TR::OptionTable OMR::Options::_feOptions[] = {
    {"gcTrace=",           "D<nnn>\ttrace gc stack walks after gc number nnn",
         TR::Options::setJitConfigNumericValue, offsetof(J9JITConfig, gcTraceThreshold), 0, "F%d"},
 #endif
+#if defined(J9VM_OPT_JITSERVER)
+   {"highActiveThreadThreshold=", " \tDefines what is a high Threshold for active compilations",
+        TR::Options::setStaticNumeric, (intptr_t)&TR::Options::_highActiveThreadThreshold, 0, " %d"},
+#endif /* defined(J9VM_OPT_JITSERVER) */
```

Comment:
`int64_t J9::Options::_highActiveThreadThreshold` is declared to be int64_t, but here it appears we want to read an intptr_t. This may break on 32-bit platforms.

Reply:
I would argue that the number of threads should fit on an int32_t

Reply:
Indeed, `int32_t getNumCompThreadsActive() const { return _numCompThreadsActive; }`